modules/ROOT/partials/configuration/a11ychecker_filter_issue.adoc

```diff
@@ -3,7 +3,7 @@
 
 The `+a11ychecker_filter_issue+` option allows Accessibility Checker issues to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `+a11ychecker_ignored_rules+` option. This option can remove issues from the results shown in the dialog and the `+getReport+` API.
 
-The function will be passed the results of the xref:a11ychecker.adoc#getreport[`+getReport()+` API]. To filter an issue from the Accessibility Checker report, the callback needs to return `+false+` for that particular issue.
+The function will be passed the results of the xref:a11ychecker.adoc#getreport[`+getReport()+` API]. To filter an issue from the Accessibility Checker report, the callback needs to return `+false+` for that particular issue. The function makes use of xref:a11ychecker.adoc#issue[`issue` objects] as the parameter for filtering issues; after they are generated by the audit, and before being displayed in the dialog.
```

Comment:
After clarifying "the results of the report api" I don't think the bit you added is necessary.

It's not clear to me whether this filter is applied during the `getReport` API function call or if integrators need to do that themselves when using the API. I didn't specify that, because I don't have time to check the code, but I think we should.

```suggestion
The callback function will be passed xref:a11ychecker.adoc#issue[`issue` objects] generated by the xref:a11ychecker.adoc#getreport[`+getReport()+` API]. To remove an issue from the Accessibility Checker report the callback needs to return `+false+` for that particular issue.
```

Reply:
Done, thanks